system/datatypes/peer_reviewed_article.php

```diff
@@ -1048,6 +1078,11 @@ private function createEditForm(array $values = []) {
 					<strong><label for="[[field-name-excerpt]]">[[excerpt]]</label></strong>[[info-button-excerpt]]<br><editor name="[[field-name-excerpt]]"></editor>
 				</div>
 			</div>
+			<div class="section" style="padding:5px; margin-bottom:5px; position:relative;">
+				<div class="input-wrapper field_type_editor field_name_[[field-name-excerpt]]">
```

Comment:
This should be featured-image rather than excerpt?

Reply:
Please: no inline styling...

Reply:
yes

Reply:
@tammoterhark : is all over the file.. I consider this to be out of the scope of this pull request.